cpp/src/ray/ray_config_internal.cc

```diff
@@ -0,0 +1,84 @@
+
+#include "gflags/gflags.h"
+
+#include "ray_config_internal.h"
+
+DEFINE_string(redis_address, "", "The ip address of redis server.");
+
+DEFINE_string(redis_password, "", "The password of redis server.");
+
+DEFINE_string(dynamic_library_path, "", "The local path of the dynamic library.");
+
+DEFINE_string(job_id, "", "Assigned job id.");
+
+DEFINE_int32(node_manager_port, 62665, "The node manager port in Raylet.");
+
+DEFINE_string(raylet_name, "", "Local raylet socket name.");
+
+DEFINE_string(object_store_name, "", "Local object store socket name.");
+
+DEFINE_string(logs_dir, "", "Log dir for workers.");
```

Comment:
This is not used, right?

Reply:
I have lose some code. It will passed from `services.py`.